Procfile

```diff
@@ -1,2 +1,2 @@
-web: bundle exec rails s
-sidekiq: bundle exec sidekiq
+web: bundle exec puma -t 5:5 -p ${PORT:-3000} -e ${RACK_ENV:-development}
```

Comment:
What is the benefit of using puma explicitly over rails s? Rails uses Puma out of the box when it is in the gemfile right? Or is that only in later rails versions?

Reply:
I think Rails does that, but I like to be explicit and avoid using Webrick by accident.

Reply:
👍